packages/decision-tree/src/DecisionTree.tsx

```diff
@@ -27,7 +30,13 @@ export const DecisionTree: FC<DecisionTreeProps> & DecisionTreeStaticMembers = (
   return (
     <Flex flexDirection="column" height="100%">
       <DecisionTree.Stepper steps={steps} activeStep={activeStep} />
-      <Question data={question} answersDispatch={answersDispatch} />
+      <Question
+        data={question}
+        answersDispatch={answersDispatch}
+        paginationDispatch={paginationDispatch}
+        defaultValue={answerState.answer}
+        submitIDKAnswer={submitIDKAnswer}
```

Comment:
we should _standardise_ the naming, it is weird to have two methods called _dispatch_ and another one submit

Reply:
I mean, 2 are dispatches and the other just a submit function, a dispatch is different than a function, it comes with an action.